data/games/under_night_in_birth_exe_late_st/movespecs/linne.json

```diff
@@ -107,24 +107,20 @@
           "6C"
         ],
         "imageId": "Wolf_Howl_1",
-        "followUps": [
-          {
-            "name": "6C",
-            "inputs": [
-              "6C"
-            ],
-            "imageId": "Wolf_Howl_2",
-            "followUps": [
-              {
-                "name": "6C",
-                "inputs": [
-                  "6C"
-                ],
-                "imageId": "Wolf_Howl_3"
-              }
-            ]
-          }
-        ]
+      },
+      {
+        "name": "6CC",
```

Comment:
I don't think these need `name` attributes anymore. That was an artifact from before we'd decided on an autocombo syntax. The movespec schema didn't allow moves with follow ups without `name` attributes.

Wouldn't it be like this?

```json
{
  "inputs": [
    "6CC"
  ],
  "imageId": "Wolf_Howl_2"
},
}
  "inputs": [
    "6CCC"
  ],
  "imageId": "Wolf_Howl_3"
}
```

See [Carmine's](https://github.com/dustup/dustup/blob/master/data/games/under_night_in_birth_exe_late_st/movespecs/carmine.json#L68-L73), [Akatsuki's](https://github.com/dustup/dustup/blob/master/data/games/under_night_in_birth_exe_late_st/movespecs/akatsuki.json#L76-L87), and [Eltnum's](https://github.com/dustup/dustup/blob/master/data/games/under_night_in_birth_exe_late_st/movespecs/eltnum.json#L86-L91) non-standard autocombos.